### ISSUE TITLE
Hyper-parameters to use?

### ISSUE THREAD

**Issue by Veronicium** [zjunlp/DocuNet#5]
Hi! I'm quite interested in your paper but I have trouble reproducing your results. When I run the run_docred.sh file, The bert-base model works fine (by reaching 61.54/59.56 F1/Ign F1, around mean-1.5sd), but my Roberta model can only get 63.30/61.40 F1/Ign F1. 

I notice some hyper-parameters listed in the script and the supplementary material is inconsistent. For example, the docred script uses bs=accum=2 but the supplementary says you use bs=5, accum=1. The supplementary says the weight decay is set to 5e-4, but I didn't see it in the code. Is this the reason why I couldn't reproduce the results?

Plus, could you upload your utils_sample file? It seems the file is missing. I plug in the one in ATLOP and that works, but I'm not sure if you write it in the same way. 

Thanks!

**Comment by TimelordRi** [comment]
Because gradient accumulation will cause a certain degree of performance degradation, it is recommended to use multiple GPU parallel training to increase the batch size.

As for the hyperparameters mentioned in supplimentray, it is because the optimized parameters forgot to update later.

Finally, the file ’utils_sample.py‘ is uploaded now. Thanks for your issues.

**Comment by Veronicium** [comment]
Thank you! I will try to use multiple GPU then
